index.html

```diff
@@ -0,0 +1,217 @@
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header class="container">
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <a href="#">My Logo</a>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+     
+        <button class="hamburger"><i class="fa fa-bars"></i></button>
+      
+    </div>
+  </nav>
+  </header>
+  <!--end of nav -->
+
+  <!--start of  section-->
+  <section class="section-wrapper">
+    <div class="section-header">
+      <h1>I'm Abdullah <br> Glad to see <br> you!</h1>
+    </div>
+    <div class="content-of-section">
+      <p>I'm a software developer!<br> I can help you build a product feature or a website. 
+        look through some of my work and experience if you like what you see and 
+        have a product you needed coded, don't hestiate to contact me.
+      </p>
+    </div>
+    
+    <div class="social-icons">
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons-list">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist"></i></a></li>
+      </ul>
+    </div>
+
+  </section> 
+      <!--work section-->
+      <section class="container-work">
+        <div class="card-info">
+        <div class="card-image">
+          <img src="images/SnapshootPortfolio0.png" alt="">
+        </div>
+        <div class="card-header">
+          <h2>Tonic</h2>
+        </div>
+        <ul class="card-details">
+          <li class="card-first-element">CANOPY</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-second-element">Back End Dev</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2015</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
+        </div>
+      </section>
+
```

Comment:
- [ ] I would suggest you make changes and stick as much as possible to Figma design.
- [ ] kindly make the `li` tag looks similar to the Figma design and makes the less spacing between the `h2` tag and the `li` tag.
- [ ] Kindly change the styling of the `li` tag (HTML, CSS, javascript) and make sure it looks like Figma design
- [ ] kindly implement all the changes in all the section
### your design
![your-design](https://user-images.githubusercontent.com/81727988/165927885-7422cb9c-e84b-4d4c-8143-ff565f8d6993.png)
## Figma design
![figma](https://user-images.githubusercontent.com/81727988/165927919-86e807f1-afe0-4442-b43e-a11299d97fbd.png)